include/qpdf/QPDFObjectHandle.hh

```diff
@@ -1852,12 +1852,6 @@ class QPDFObjectHandle::QPDFArrayItems
     QPDFObjectHandle oh;
 };
 
-inline QPDFObjGen
-QPDFObjectHandle::getObjGen() const
-{
-    return isInitialized() ? obj->getObjGen() : QPDFObjGen();
-}
-
 inline int
```

Comment:
Given that we can't inline getObjGen, it probably doesn't make sense to inline getObjID , getGeneration and isIndirect either. (If my understanding is correct, either way we end up with one function call, and if the methods are not inline, marginally fewer instructions at the point of call.)

Reply:
If you want to try it and see, feel free, and I can merge when I'm back. Otherwise I'll try tomorrow or Tuesday.

Reply:
On my system, removing these inlines actually results in a measurable performance decrease, so I think leaving them as they are is probably better. This may be dependent on the compiler or flags, but the way I build it is similar to the way it's built by most distributions, so I think I'll leave it alone.